test/sql/insert_single.sql

```diff
@@ -21,6 +21,14 @@ INSERT INTO "1dim" SELECT * FROM regular_table;
 SELECT * FROM "1dim";
 SELECT "1dim" FROM "1dim";
 
+--test that we can insert pre-1970 dates
+CREATE TABLE "1dim_pre1970"(time timestamp PRIMARY KEY, temp float);
+SELECT create_hypertable('"1dim_pre1970"', 'time');
+INSERT INTO "1dim_pre1970" VALUES('1965-01-20T09:00:21', 21.2);
+INSERT INTO "1dim_pre1970" VALUES('1965-01-20T09:00:47', 25.1);
+SELECT * FROM "1dim_pre1970";
+
```

Comment:
I think we need more tests here.  Can we insert stuff on both sides of the 1970 boundary? Also can we view the resulting chunks and their check constraints with `\d+ <chunk>`?